### DISCUSSION TITLE
Allow to add tags/flags to orders

### DISCUSSION THREAD

**Discussion by Unicorn-rzl** [pretix/pretix#1960] [Ideas]
I had the idea to allow the admin to add tags to orders (sth. like flags) which the admin can use to document the completion of certain tasks regarding that order. 
I would have used it during this years MRMCD to track the state of the orders requiring sending out some stuff, but it could also be used to document things like "badge for this order printed" or what ever else future organizers come up with.

**Comment by rixx** [comment]
We could include this in the re-write of Voucher Tags - implementing one tagging system. Then we'd have the availability to add multiple tags to any voucher and any order.

**Comment by rixx** [comment]
Hm, at least we have order comments by now, which have some of the use case, but cannot be filtered or sorted for. Opinions?
